app/components/message-list/MessageList.js

```diff
@@ -1,7 +1,7 @@
 (function () {
     'use strict';
 
-    var module = angular.module('chatApp.MessageList', ['chatApp.DataModel']);
+    var module = angular.module('chatApp.MessageList', ['chatApp.DataModel', 'dbaq.emoji', 'ngSanitize']);
         
     module.controller('MessagesController', function($scope, DataModelService) {
             $scope.messageList = 'List of Messages';
```

Comment:
Don't move constants to controller. Feel free to hardcode them inside templates. If you would like to support more then 1 language, then take a look at angular-translate module.